lib/kino/explorer.ex

```diff
@@ -121,6 +137,7 @@ defmodule Kino.Explorer do
 
   defp type_of(dtype, _) when dtype in [:integer, :float], do: "number"
   defp type_of(dtype, _) when dtype in [:date, :datetime], do: "date"
+  defp type_of(:boolean, _), do: "boolean"
   defp type_of(:string, [data]), do: type_of_sample(data)
   defp type_of(_, _), do: "text"
```

Comment:
WDYT about:

```suggestion
  defp type_of(:string, [data]), do: type_of_sample(data)
  defp type_of(type, _), do: Atom.to_string(type)
```

Reply:
What I’m doing here is pairing the data type with the Glide cell kind. That’s the reason I’m using the Glide naming. We can think about a better solution but just using the dtype won’t work. And if we decide to change, we need to assure that the DataTable and the front made the changes too and keep all on the same page!